api/src/main/java/ai/djl/modality/cv/translator/BigGANTranslator.java

```diff
@@ -40,13 +40,15 @@ public BigGANTranslator(float truncation) {
     /** {@inheritDoc} */
     @Override
     public Image[] processOutput(TranslatorContext ctx, NDList list) {
-        NDArray output = list.get(0).addi(1).muli(128).clip(0, 255);
+        Image[] images;
+        try (NDArray output = list.get(0).duplicate().addi(1).muli(128).clip(0, 255)) {
```

Comment:
use try for NDArray makes the code looks ugly.
We generally don't use try-with-resource in Translator.

Reply:
got it - i'll update